cmd/cerca/cerca.go

```diff
@@ -11,6 +11,36 @@ import (
 	"cerca/util"
 )
 
+func createHelpString(usageExamples []string, isMainCmd bool) string {
+	helpString := fmt.Sprintf("USAGE:\n    %s\n", strings.Join(usageExamples, "\n    "))
+
+	if isMainCmd {
+		helpString += "\nCOMMANDS:" + `
+    admin     promote user to admin
+    migrate   manage database migrations
+    user      create new user
+    reset     reset user password` + "\n"
```

Comment:
in the shower (of course, where else) i thought that maybe the naming of the commands could be improved.

`cerca admin` doesn't tell enough about its function, while `cerca make-admin` does. following that line of thought, what do you think about the following command names? and to also reorder them based on alphabetic order, also below

1. user -> adduser 
2. admin -> makeadmin
3. migrate -> migrate
4. reset -> resetpw

Reply:
actually fuk it i'm just going to merge and do some last minute tidying up as a way to tie the bow on this super refactor! thank you d1 :muscle: :heart: